tests/zib_uploader/test_fetch_data.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+from zib_uploader.fetch_data import ClassMappingsFetcher, OntologyClassMapping
+
+
+@pytest.mark.integration_test
```

Comment:
For me this is the most straightforward way to test this. But we do not want to run these kind of tests in an automated testing pipeline. I am used to marking that like this (this makes it easy to ignore these later).

An alternative could be to write a unittest and mock the bioportal API, but that is actually the main thing I am interested in for testing...